etip/trackers/management/commands/compare_with_exodus.py

```diff
@@ -0,0 +1,99 @@
+
+    def handle(self, *args, **options):
+        exodus_trackers = self.get_all_from_exodus()
+        self.stdout.write(
+            'Retrieved {} trackers from Exodus'.format(len(exodus_trackers))
+        )
+        self.count_from_etip()
+        self.lookup_trackers(exodus_trackers)
+
+    def __exact_lookup(self, tracker_details):
+        for idx, field in enumerate(FIELDS_TO_LOOKUP):
+            kwargs = {
+                'is_in_exodus': True,
+                field + '__iexact': tracker_details.get(field)}
+            results = Tracker.objects.filter(**kwargs)
+            if not results and idx + 1 == len(FIELDS_TO_LOOKUP):
+                return (LOOKUP_NOT_FOUND, None)
+            elif not results:
+                continue
+            elif len(results) != 1:
+                return (LOOKUP_MULTIPLE, results)
+            else:
+                result = results[0]
+                remaining_fields = FIELDS_TO_LOOKUP.copy()
+                remaining_fields.remove(field)
+                for field in remaining_fields:
+                    if getattr(result, field) != tracker_details.get(field):
+                        return (LOOKUP_IS_DIFFERENT, result)
+                return (LOOKUP_IS_EXACT, result)
+
+    def lookup_trackers(self, exodus_trackers):
+        results_count = {
+            LOOKUP_IS_EXACT: 0,
+            LOOKUP_IS_DIFFERENT: 0,
+            LOOKUP_MULTIPLE: 0,
+            LOOKUP_NOT_FOUND: 0
+        }
+        self.stdout.write('Starting case-insensitive lookup...')
+        for id, tracker_details in exodus_trackers.items():
+            lookup_result = self.__exact_lookup(tracker_details)
+            if lookup_result[0] == LOOKUP_IS_EXACT:
+                results_count[LOOKUP_IS_EXACT] += 1
+                continue
+            elif lookup_result[0] == LOOKUP_IS_DIFFERENT:
+                results_count[LOOKUP_IS_DIFFERENT] += 1
+            elif lookup_result[0] == LOOKUP_MULTIPLE:
+                results_count[LOOKUP_MULTIPLE] += 1
+            elif lookup_result[0] == LOOKUP_NOT_FOUND:
+                results_count[LOOKUP_NOT_FOUND] += 1
+            self.stdout.write(
+                '{} - {}'.format(
+                    lookup_result[0], tracker_details.get('name')
+                )
+            )
+        self.stdout.write('Lookup results:')
+        for type in results_count:
+            self.stdout.write('** {}: {}'.format(type, results_count[type]))
+
+    def get_all_from_exodus(self):
+        response = requests.get(EXODUS_API_BASE_URL + EXODUS_API_BASE_PATH)
+        if response.status_code != 200:
+            raise CommandError(
+                'Unexpected status from API: {}'
+                .format(response.status_code)
+            )
+        resp = response.json()
+        if resp is None or resp.get('trackers') is None:
+            raise CommandError('Empty response')
+        return resp.get('trackers')
+
+    def count_from_etip(self):
```

Comment:
The point of this function is only to print the number of trackers?
If so, I would:
- remove the unnecessary "return" at the end
- do `in_exodus_trackers_nb = Tracker.objects.filter(is_in_exodus=True).count()`